finstmt/config_manage/statement.py

```diff
@@ -33,6 +33,19 @@ def get(self, item_key: str) -> ItemConfig:
                 continue
         raise NoSuchItemException(item_key)
 
+    def __getattr__(self, item_key: str) -> ItemConfig:
+        """
+        Get the config for a given key
+        """
+        try:
```

Comment:
I thought this would be fine, but it looks like from the tests that this breaks copying the statements. `__getattr__` is called when the attribute is not otherwise defined. This means that there's no reason normally that `get` should be able to call `__getattr__` because the only attribute access is `config_managers`, which should be defined. But it seems like during copying, the object is in some partial initialization state where `config_managers` has not been assigned yet. Therefore it calls `__getattr__` to resolve `config_managers` during `.get`, which then again calls `.get` and we end up with a recursive loop until it crashes the program.

I think this will fix it, can you try committing and we'll see if tests pass? 
```suggestion
        # When copying the object, config_managers can be undefined and so would cause a 
        # recursive loop without this check
        if item_key == "config_managers":
            # Trigger the default Python behavior
            return object.__getattribute__(self, item_key)
        
        try:
```